common/display/virtualdisplay.cpp

```diff
@@ -150,6 +150,7 @@ bool VirtualDisplay::Present(std::vector<HwcLayer *> &source_layers,
                              bool handle_constraints) {
 #ifdef HYPER_DMABUF_SHARING
   if (display_index_ == 0) {
+    CTRACE();
```

Comment:
Why add this line?
I think it is not aligned with the commit title, right
Suggest to add in another commit if you feel need